src/test/java/org/folio/moduserstest/RestVerticleIT.java

```diff
@@ -89,61 +91,42 @@
     .put("requestForSponsor", "Yes")
     .put("notificationsTo", "Proxy")
     .put("accrueTo", "Sponsor");
-
-
+  
+  private static Vertx vertx;
+  
   @Rule
   public Timeout rule = Timeout.seconds(20);
 
   @BeforeClass
   public static void setup(TestContext context) {
-    RestITSupport.setUp();
+    vertx = Vertx.vertx();
 
-    Async async = context.async();
-    TenantClient tenantClient = new TenantClient(RestITSupport.HTTP_LOCALHOST + RestITSupport.port(), "diku", FAKE_TOKEN);
+    Integer port = NetworkUtils.nextFreePort();
+    RestITSupport.setUp(port);
+    TenantClient tenantClient = new TenantClient("http://localhost:" + Integer.toString(port), "diku", "diku");
     DeploymentOptions options = new DeploymentOptions()
-      .setConfig(new JsonObject().put("http.port", RestITSupport.port()))
-      .setWorker(true);
-
-    RestITSupport.vertx().deployVerticle(RestVerticle.class.getName(), options, context.asyncAssertSuccess(res -> {
-      // remove existing schema from previous tests
-      tenantClient.deleteTenant(delete -> {
-        switch (delete.statusCode()) {
-          case 204: break;  // existing schema has been deleted
-          case 400: break;  // schema does not exist
-          default:
-            RestITSupport.fail(context, "deleteTenant", delete);
-            return;
-        }
-        try {
-          TenantAttributes ta = new TenantAttributes();
-          ta.setModuleTo("mod-users-1.0.0");
-          List<Parameter> parameters = new LinkedList<>();
-          parameters.add(new Parameter().withKey("loadReference").withValue("true"));
-          parameters.add(new Parameter().withKey("loadSample").withValue("false"));
-          ta.setParameters(parameters);
-          tenantClient.postTenant(ta, post -> {
-            if (post.statusCode() != 201) {
-              RestITSupport.fail(context, "postTenant", post);
-            }
-            async.complete();
-          });
-        } catch (Exception e) {
-          context.fail(e);
-        }
-      });
+        .setConfig(new JsonObject().put("http.port", port));
+
+    vertx.deployVerticle(RestVerticle.class.getName(), options, context.asyncAssertSuccess(res -> {
+      TenantAttributes ta = new TenantAttributes();
+      ta.setModuleTo("mod-users-1.0.0");
```

Comment:
@julianladisch @adamdickmeiss Does the `toModule` property still need to be set when invoking the Tenant API?

I thought RAML Module Builder now tracked versions internally.

Reply:
It should be there to indicate create (both from + to to indicate upgrade). RMB will not care about the values. But others might.

Reply:
Thanks @adamdickmeiss for clarifying those expectations.

@K-Felk please could a comment be added to explain that the version doesn't matter to RAML Module Builder, and this is used as a marker for a new activation rather than an upgrade.

Reply:
This is done, you may want to check the comment to make sure it reflects this discussion accurately.